docs/Consuming.md

```diff
@@ -0,0 +1,14 @@
+# Consuming
+
+Consuming topics can be done by right clicking a topic in the explorer or from the command palette.  Some things to note about consuming:
+
+* UTF-8 encoded keys and values only. If data is encoded differently, it will not be pretty.
+* One consumer group is created per topic (may change in the future to just have one for the extension).
+
+Consumers are based on virtual documents, available in the VS Code extension API. A consumer will keep running even if you close the document in the editor. You should make sure to close the consumer explicitly, either via the command palette, the status bar element or the start/stop action button as well. The VS Code API does not support detecting if a virtual document is closed immediately. Instead, the underlying virtual document is automatically closed after two minutes if the document is closed in the editor.
+
+# Preferences
+
+## kafka.consumers.offset

```

Comment:
To make the preference keys more readable, I found that using this pattern works better:

``` 
### `kafka.consumers.offset`
```

Reply:
fixed